### ISSUE TITLE
Add packaging guidelines

### ISSUE THREAD

**Issue by matteodelabre** [toltec-dev/toltec#18]
- [x] Description of the ipk format.
- [x] Description of the recipe format.
- [x] Choosing a correct version number.
- [x] Choosing a section.
- [x] How to build and test a package.

**Comment by LinusCDE** [comment]
I created a guide for packaging [here](https://gist.github.com/LinusCDE/ae13a29a90b8de363d9724c9d075d909). Looking at `doc/package.md` it describes the file format. We should consider merging those two files into one for finding some other useful name for this file.

I refrained from doing that now since the guide probably not not that clear yet and I might mess it up just more.

To encourage quick changes I sent a link to a editable version on hackmd in discord [here](https://discordapp.com/channels/385916768696139794/756853330382422037/759822617850544188). Imo this is faster than having a pull request with a lot of suggested changes.

**Comment by LinusCDE** [comment]
The gist above should be continued on the branch `patch-maintainer-scripts`. The above mentioned file is there now as [docs/package_guide.md](https://github.com/toltec-dev/toltec/blob/patch-maintainer-scripts/docs/package_guide.md)

**Comment by LinusCDE** [comment]
Is now moved to PR #76 . Please do any changes regarding the documentation there instead.

**Comment by Eeems** [comment]
@matteodelabre Is the checklist here up to date?

**Comment by matteodelabre** [comment]
I just updated it. Thanks!

**Comment by Eeems** [comment]
For the first item, do we need to describe the ipk format, or just link to a good resource?

**Comment by Eeems** [comment]
Looks like we have one item left on this. I'm guessing this is waiting on work in https://github.com/toltec-dev/build?

**Comment by matteodelabre** [comment]
Well I’ve already written some documentation on how to build & test packages in that repo’s readme & docs folder. Do you have more items in mind that you would want to see covered in the docs?

**Comment by Eeems** [comment]
Not really, after taking a peek at the repo's readme I think we can probably call this resolved.

**Comment by matteodelabre** [comment]
The remaining work would be to remove the duplicate docs from this repo and link to the other repo’s documentation.

**Comment by Eeems** [comment]
@matteodelabre should that be done as part of this issue or a new one? This is the last issue outstanding before we can resolve #15  and close the community rollout milestone.

**Comment by matteodelabre** [comment]
Yes, I think we can close this. There is already a (quite basic) guide on how to build and test a package here: <https://github.com/toltec-dev/build/blob/main/docs/writing-recipes.md#testing-your-package>. We should track further improvements to this guide in a separate issue.
